crosswalks/Composer.csv

```diff
@@ -0,0 +1,68 @@
+Property,Packagist
+codeRepository,support.source
+programmingLanguage,
+runtimePlatform,
+targetProduct,
+applicationCategory,type
+applicationSubCategory,
+downloadUrl,
+fileSize,
+installUrl,
+memoryRequirements,
+operatingSystem,
+permissions,
+processorRequirements,
+releaseNotes,
+softwareHelp,
+softwareRequirements,
+softwareVersion,version
+storageRequirements,
+supportingData,
+author,authors
+citation,
+contributor,
+copyrightHolder,
+copyrightYear,
+dateCreated,
+dateModified,
+datePublished,
+editor,
+encoding,
+fileFormat,
+funder,
+keywords,keywords
+license,license
+producer,
+provider,
+publisher,
+sponsor,
+version,version
+isAccessibleForFree,
+isPartOf,
+hasPart,
+position,
+description,description
+identifier,name

```

Comment:
I'm not sure we want this, names are generally not good identifiers (though I realize https://github.com/codemeta/codemeta/blob/master/crosswalks/NodeJS.csv has the same issue)

Reply:
Should I take it out then?
(I added it just because it was in the file you linked)

Reply:
@cboettig & @mbjones: we need your expertise here